packages/core/strapi/bin/strapi.js

```diff
@@ -303,6 +304,12 @@ if (process.env.STRAPI_EXPERIMENTAL === 'true') {
         () => exitWith(1, 'At least one source (from) or destination (to) option must be provided')
       )
     )
+    .hook(
+      'preAction',
+      confirmMessage(
+        'The import will delete all data in the remote database. Are you sure you want to proceed?'
```

Comment:
Don't we need to consider the strategy that is being issued for future messages? What do you think?

Reply:
Once we have other strategies, yeah. In fact it used to check, but it was getting overly complicated for no reason considering we only had the one :)

Reply:
Yes that makes sense